### PR TITLE
Use internal computed-model identifiers for entry_id in containter_identifiers

### PR DESCRIPTION
**Pull request by piehld**

### CODE REVIEW

**Review by brindakv — COMMENTED** [review]
@piehld Looks good. Just one clarification. Please see my comment.

File: rcsb/utils/dictionary/DictMethodCommonUtils.py [hunk 1]
```diff
@@ -1089,7 +1090,7 @@ def __getAtomSiteInfo(self, dataContainer, modelId="1"):
         brAuthAsymIdMapD = {}
         try:
             eObj = dataContainer.getObj("entry")
-            entryId = eObj.getValue("id", 0)
+            sourceEntryId = eObj.getValue("id", 0)
```
[the remark this answers is not in the document]
Comment (piehld) [reply rcsb/utils/dictionary/DictMethodCommonUtils.py:1093]: If you look further down in that method, I think you'll see why I had to do this. Basically the `entryId` needs to be used as a temp variable, so I created the `sourceEntryId` as a permanent variable to store the true entry Id.

[the remark this answers is not in the document]
Comment (piehld) [reply rcsb/utils/dictionary/DictMethodCommonUtils.py:1093]: (You'll need to look at it as the complete file, not just showing the changes. You'll see what I mean)

[the remark this answers is not in the document]
Comment (brindakv) [reply rcsb/utils/dictionary/DictMethodCommonUtils.py:1093]: I looked at the entire method. What I am trying to clarify is whether `entryId` can be retained as the permanent variable and set to `_entry.id` for experimental models and `_entry.rcsb_comp_model_id` for computed models? Is there a need for `_entry.id` for computed models in this context?

[the remark this answers is not in the document]
Comment (piehld) [reply rcsb/utils/dictionary/DictMethodCommonUtils.py:1093]: Ah, I think I see what you mean. We can just reassign `entryId` to the computed-model ID at the top of the method, and remove all the "isComputedModel" conditionals later on down. Yes that's a much cleaner fix. Will push in a second, thanks.

**Review by brindakv — APPROVED** [review]
@piehld Looks good to me.

**Pull request merged**